File: Leetcode-Solutions/Arrays/TwoSum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        differ={}
        for i in range(len(nums)):
            if target-nums[i] in differ:
                return [differ[target-nums[i]],i]
            differ[nums[i]]=i
        return []

File: Leetcode-Solutions/Arrays/twoSum.ts
function twoSum(nums: number[], target: number): number[] {
    let diff={}
    let res=[]
    let i=0
    while (i<nums.length){
      if(diff.hasOwnProperty(target-nums[i]) ){
        res.push(diff[target-nums[i]])
        res.push(i)
        break
      }
     
      diff[nums[i]]=i;
       i++
      }

      return res

